ui_tcell.go

```diff
@@ -119,9 +121,12 @@ func (ui *tcellUI) handleEvent(ev event) {
 		ui.root.OnKeyEvent(e)
 		ui.painter.Repaint(ui.root)
 	case callbackEvent:
+		// Gets stuck in a print loop when the logger is a widget.
+		//logger.Printf("Received callback event")
```

Comment:
Having callbacks unlogged misses a whole class of sequencing events- do we really want to miss all of these in order to support the inband logging widget?


One way to support the inband widget without this suppressing all callback logging: Add a 'silent' attribute to `callbackEvent`, and introduce a package-private `silentUpdate(func())` which sends a `callbackEvent` with it set. Provide the `Logger` widget which uses that instead of Update when setting its contents.